server/api/users.js

```diff
@@ -221,3 +221,18 @@ router.put('/editPrivilege', async (req, res, next) => {
     next(err);
   }
 });
+
+
+router.put('/editEmail', async (req, res, next) => {
+  try {
+    const user = await User.findByToken(req.headers.authorization);
+
```

Comment:
@kuperavv can you review this section later if you spruce up restful routes so that user auth will throw error or be used properly as prof expects